plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/FileWithModifiedLines.java

```diff
@@ -0,0 +1,58 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * Model class containing data pertaining to files with modified lines of code. Each object contains a relative file
+ * path and a list of {@link ModifiedLinesBlock} objects.
+ */
+@ExportedBean
+public class FileWithModifiedLines {
+    private final String relativePath;
+    private final List<ModifiedLinesBlock> listOfModifiedLines;
+
+    /**
+     * Constructor for the FileWithModifiedLines class.
+     *
+     * @param relativePath
+     *         the relative path of the file
+     * @param listOfModifiedLines
+     *         all modified lines inside the file
+     */
```

Comment:
Either make package private (if possible, see above) and remove the comment. Or use something like:
```suggestion
    /**
     * Creates a new instance of {@link FileWithModifiedLines}.
     *
     * @param relativePath
     *         the relative path of the file
     * @param listOfModifiedLines
     *         all modified lines inside the file
     */
```

Reply:
Done. Files have been made package private if possible, otherwise the correct format is being used.